### DISCUSSION TITLE
Hansel sap conguration per project

### DISCUSSION THREAD

**Discussion by jeusdi** [LazyVim/LazyVim#2197] [Q&A]
I know I need to add a dap-configuration in order to attach my debugging:

```
local dap = require('dap')
dap.configurations.java = {
  {
    type = 'java';
    request = 'attach';
    name = "Debug (Attach) - Remote";
    hostName = "127.0.0.1";
    port = 5005;
  },
}
```

Two questions:
1. Is there any way to create above dap-configuration on the fly?
2. Is there any way to load dap-configurations per project? I was thinking on [neoconf.nvim](https://github.com/kauffinger/lazyvim/blob/main/.neoconf.json).

**Comment by dpetka2001** [comment]
Regarding question 2, take a look at #2115. You might find it useful.
